docs/_toc.yaml

```diff
@@ -35,6 +35,8 @@ toc:
   path: /cirq/experiments/quantum_chess
 - title: "Concepts"
   path: /cirq/experiments/quantum_chess/concepts
+  title: "Quantum Chess REST API"
+  path: /cirq/experiments/quantum_chess/Quantum_Chess_REST_API
```

Comment:
```suggestion
  title: "Quantum chess REST API"
  path: /cirq/experiments/quantum_chess/quantum_chess_rest_api
```

Reply:
Quantum Chess is the title of the game, I'm not sure it is appropriate to make that lower case. It would be like saying World of warcraft. Let me know.

Reply:
Changed file names, and moved images.

Reply:
Ok, "Quantum Chess" makes sense. Thanks.
I noticed some inconsistencies with naming in the other docs. May need to do a scrub.

Reply:
Is this good to squash and merge? I looked through the rest of the files and didn't see anything egregious.

Reply:
Hey Chris. Title is fine but please lower-case the filenames

Reply:
Updated file names and docs/_toc.yaml